test/AeroStrategy.js

```diff
@@ -0,0 +1,437 @@
+const {
+    loadFixture,
+    mine,
+    time,
+    reset,
+} = require("@nomicfoundation/hardhat-network-helpers");
+const { ZERO_ADDRESS } = require("@openzeppelin/test-helpers/src/constants");
+const { expect } = require("chai");
+const { utils, BigNumber } = require("ethers");
+const { parseEther } = require("ethers/lib/utils");
+const { ethers } = require("hardhat");
+
+const { getEnv } = require("../scripts/utils");
+
+const IERC20_SOURCE = "@openzeppelin/contracts/token/ERC20/IERC20.sol:IERC20";
+
+const BASE_NODE = getEnv("BASE_NODE");
+const BASE_FORK_BLOCK = getEnv("BASE_FORK_BLOCK");
+
+// upgrades.silenceWarnings();
+
+describe("AeroStrategy", function () {
```

Comment:
Basically okay, just have issues like mixing fixturing and test state definition with an actual test case. Need to be rewritten in the possible future. And also logging - it does not use verbosity levels and has monkey business like `console.log(1)`.